test_data.sql

```diff
@@ -2,8 +2,14 @@ DELETE FROM CourseEnrollments;
 DELETE FROM Courses;
 DELETE FROM Students;
 
-INSERT INTO Students(id, first_name, last_name) VALUES (3, "Giulia", "Contarino"), (2, "Tommaso", "Tranchese"), (1, "Federico", "Urbani");
+INSERT INTO Students(id, first_name, last_name) VALUES (3, "Giulia", "Contarino"), (2, "Tommaso", "Tranchese"), (1, "Federico", "Urbani"), (4, "Lucrezia", "Rumore");
```

Comment:
anche questi, se aggiungi dati di test in più crea un file a parte sotto la tua cartella personale